src/node_api.cc

```diff
@@ -3088,6 +3088,18 @@ napi_status napi_get_version(napi_env env, uint32_t* result) {
   return napi_clear_last_error(env);
 }
 
+napi_status napi_get_node_version(napi_env env,
+                                  uint32_t* versions,
+                                  const char** release) {
```

Comment:
> but as a developer it may not be clear what the scope of the string is and whether it needs to be freed by the caller.

I can add a bit to the docs if you have a suggestion for wording, but since this refers to a **const** char pointer, there isn’t really any ambiguity.